src/scripts/app.js

```diff
@@ -1 +1,22 @@
 console.log('hello');
+
+{
+  // Header main menu open animation
+
+  const mobileMainMenuTrigger = document.querySelector('.mobile-menu-trigger-button');
+  const mobileMainMenu = document.querySelector('.menu-bar');
+  const mobileMainMenuActiveItem = document.querySelector(
+    '.menu > ul > li:first-child a'
+  );
+
+  const showMainMenu = () => {
```

Comment:
showMainMenu is a reference to your functional expression.
create funcion or move your funcional expression to EventListener.

Reply:
showMainMenu is ES6 arrow function declaration.
Keepeing code readaibility in mind, imho, it is better to first declare function and then bind it to eventListenter.

Reply:
resolved

Reply:
you don't declared function, you declared reference to functional expression. it is a difference.